src/app/modules/form/admin-form/admin-form.middlewares.ts

```diff
@@ -40,13 +40,23 @@ export const updateSettingsValidator = celebrate({
     payments_field: Joi.object({ gst_enabled: Joi.boolean() }),
     workflow: Joi.array()
       .items(
-        Joi.object().keys({
+        Joi.object({
           _id: Joi.string(),
           workflow_type: Joi.string().valid(...Object.values(WorkflowType)),
-          emails: Joi.alternatives().try(
-            Joi.array().items(Joi.string().email().allow('')),
-            Joi.string().email({ multiple: true }).allow(''),
-          ),
+          emails: Joi.when('workflow_type', {
+            is: WorkflowType.Static,
+            then: Joi.alternatives()
+              .try(
+                // We might want to tighten this validation
+                Joi.array().items(Joi.string().email().allow('')),
+                Joi.string().email({ multiple: true }).allow(''),
```

Comment:
Tightening as in making this to not allow empty strings?

Reply:
Seems like from the schema, we should only accept `string[]`

Reply:
Yeah, by right only should accept `string[]`, but there is a `transformEmails` step in the mongoose schema which does the following:

`/src/app/modules/form/form.utils.ts`

```
// Converts 'test@hotmail.com, test@gmail.com' to ['test@hotmail.com', 'test@gmail.com']
export const transformEmailString = (v: string): string[] => {
  return v
    .split(',')
    .map((item) => item.trim().toLowerCase())
    .filter((email) => email.includes('@')) // remove ""
}

// Function that coerces the string of comma-separated emails sent by the client
// into an array of emails
export const transformEmails = (v: string | string[]): string[] => {
  // Cases
  // ['test@hotmail.com'] => ['test@hotmail.com'] ~ unchanged
  // ['test@hotmail.com', 'test@gmail.com'] => ['test@hotmail.com', 'test@gmail.com'] ~ unchanged
  // ['test@hotmail.com, test@gmail.com'] => ['test@hotmail.com', 'test@gmail.com']
  // ['test@hotmail.com, test@gmail.com', 'test@yahoo.com'] => ['test@hotmail.com', 'test@gmail.com', 'test@yahoo.com']
  // 'test@hotmail.com, test@gmail.com' => ['test@hotmail.com', 'test@gmail.com']
  if (Array.isArray(v)) {
    return v.flatMap(transformEmailString)
  } else {
    return transformEmailString(v)
  }
}
``` 

So it seems actually that it CAN take string-separated values, but we should probably just disallow that in general. Will fix this!

Reply:
done in [de93eb9](https://github.com/opengovsg/FormSG/pull/7078/commits/de93eb964d8d9259cc4f59717c55815e2936957d)